tests/Oryx.Integration.Tests/Database/PulledBuildOsTypeTest.cs

```diff
@@ -0,0 +1,74 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Microsoft.Oryx.Tests.Common;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Oryx.Integration.Tests
+{
+    [Trait("category", "db")]
+    public class PulledBuildOsTypeTest
```

Comment:
Based this test heavily on https://github.com/microsoft/Oryx/blob/main/tests/Oryx.Integration.Tests/Database/PulledBuildImageTypeTest.cs